charts/lagoon-core/values.yaml

```diff
@@ -3,6 +3,7 @@
 # elasticsearchHost:
 # harborURL:
 # harborAdminPassword:
+# harborAPIVersion:
```

Comment:
can we have a default on harbor api v2? and then just if somebody really wants to use v1 they can change the apiversion?

Reply:
Great idea. I committed that change.